frontend/app/rollup.config.mjs

```diff
@@ -163,6 +163,15 @@ function transformSourceMappingUrl(contents) {
         .replace("//# sourceMappingURL=", "//# sourceMappingURL=./_/raw/");
 }
 
+function watchForWorkerChanges() {
```

Comment:
By default only files in the module graph are included, this instructs Rollup to additionally watch for changes to the worker.